src/stopwords.c

```diff
@@ -76,12 +81,12 @@ StopWordList *NewStopWordListCStr(const char **strs, size_t len) {
     }
     size_t tlen = strlen(t);
 
-    // lowercase the letters
-    for (size_t pos = 0; pos < tlen; pos++) {
-      if (isalpha(t[pos])) {
-        t[pos] = tolower(t[pos]);
-      }
+    // convert multi-byte characters to lowercase
+    size_t newLen = unicode_tolower(t, tlen);
+    if (newLen) {
+      tlen = newLen;
     }
+
```

Comment:
Is this the same semantics?
What happens if there are numbers inside, that were skipped before?

Reply:
I suppose digits will remain digits, no?